lib/peep.rb

```diff
@@ -0,0 +1,47 @@
+require 'pg'
+
+class Peep
+  attr_reader :id, :peep, :user, :created_at
+
+  def initialize(id, peep, user, created_at)
+    @id = id
+    @peep = peep
+    @user = user
+    @created_at = created_at
+  end
+
+  def self.post(options)
```

Comment:
This is a minor issue, but conventionally the method for adding resources to a db would be called create and the method for displaying all instances of a resource (on line 23) would be called all.

This is just convention, and the names you've given in place of these are perfectly clear, but generally speaking following convention helps to reveal your intentions which is always a good thing.